srsue/hdr/stack/upper/nas_config.h

```diff
@@ -67,6 +67,8 @@ class nas_5g_args_t
   std::string                    ia5g;
   std::string                    ea5g;
   std::vector<pdu_session_cfg_t> pdu_session_cfgs;
+  bool emergency_registration;
```

Comment:
this is for 5g only, so maybe call it `emergency_registration_5g`?

Reply:
FIxed, changed it to emergency_registration_5g.